src/radical/pilot/utils/prof_utils.py

```diff
@@ -83,10 +83,10 @@
 
       # # if we have cmd_start / cmd_stop:
       # 'exec_sh'     : [{ru.EVENT: 'cu_start'               },
-      #                  {ru.EVENT: 'cmd_start'              }],
-      # 'exec_cmd'    : [{ru.EVENT: 'cmd_start'              },
-      #                  {ru.EVENT: 'cmd_stop'               }],
-      # 'term_sh'     : [{ru.EVENT: 'cmd_stop'               },
+      #                  {ru.EVENT: 'app_start'              }],
+      # 'exec_cmd'    : [{ru.EVENT: 'app_start'              },
+      #                  {ru.EVENT: 'app_stop'               }],
+      # 'term_sh'     : [{ru.EVENT: 'app_stop'               },
```

Comment:
Why the change in naming? In our lab parlance, `application` refers to the user-facing application while `command` to the executable of a task. While I do not care about our parlance, I care about consistency,

Reply:
Apologies, this came from a merge, and I resolved this apparently the wrong way.  Thanks for catching, will fix!